toy_robot.rb

```diff
@@ -1,24 +1,25 @@
 require_relative "invalid_position_error"
 
 class ToyRobot
-  attr_reader :x, :y, :f, :movements
+  attr_reader :x, :y, :facing, :movements
 
   def initialize
     @x = 0
     @y = 0
-    @f = :east
+    @facing = :north
   end
 
-  def place(x,y)
-    raise InvalidPositionError.new("Invalid Position, #{x}, #{y}") if (x<0 || y<0)
+  def place(x,y,facing)
+    raise InvalidPositionError.new("Invalid Position, #{x}, #{y}, #{facing}") if (x<0 || y<0 || x>4 || y>4)
```

Comment:
There is nothing wrong with this code but you could make it more readable by using https://ruby-doc.org/core-2.2.0/Comparable.html#method-i-between-3F ?